backend/server/app.py

```
@@ -12,6 +12,7 @@
 import asyncio
 import logging
 from decimal import Decimal
+from core.CalibrationEngine import CalibrationEngine, SurfaceCalibrationResult
```

Ruff (0.12.2)

`core.CalibrationEngine.SurfaceCalibrationResult` imported but unused

Remove unused import: `core.CalibrationEngine.SurfaceCalibrationResult`

(F401)